kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -566,6 +572,22 @@ public SchemaRetriever getSchemaRetriever() {
     return schemaRetriever;
   }
 
+  /**
+   *
+   * @return An optional class of KafkaKeyFieldName.
```

Comment:
Nit: These are not really useful, and not quite accurate. Something like:

```
If the connector is configured to load Kafka keys into BigQuery, this field defines
what the name of the field is. A structure is created under the field name to contain
a topic's Kafka key schema.
@return Field name for Kafka keys to be used in BigQuery
```

Reply:
Thanks for the suggestion! Updated accordingly.